TestAssets/LockFileSamples/ExportFileSamples/valid/project.fragment.lock.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
The use of `valid` and `invalid` seems inappropriate for test assets. These are projects that exhibit some characteristics. Those characteristics' validity is really defined in tests. I'd have an easier time debugging tests if the projects describe the scenarios they are representing and the test code told me if/why valid.